docs/evaluation/how_to_guides/llm_as_judge.mdx

```diff
@@ -1,24 +1,36 @@
+
 import {
   CodeTabs,
   python,
   typescript,
 } from "@site/src/components/InstructionsWithCode";
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import '@site/src/css/custom.css';
+
+
 # How to define an LLM-as-a-judge evaluator
 
 :::info Key concepts
+- [LLM-as-a-judge evaluator](/evaluation/concepts#llm-as-judge)
+:::
 
-- [LLM-as-a-judge evaluator](../concepts#llm-as-judge)
+LLM applications can be challenging to evaluate since they often generate conversational text with no single correct answer.
 
-:::
+This guide shows you how to define a LLM-as-a-judge evaluator for [offline evaluation](/evaluation/concepts#offline-evaluation) using either the LangSmith SDK or the UI. Note: To run evaluations in real-time on your production traces, refer to [setting up online evaluations](/observability/how_to_guides/online_evaluations#configure-llm-as-judge-evaluators). 
+
+<Tabs className="interface-tabs">
+  <TabItem value="sdk" label="SDK">
 
-LLM applications can be difficult to evaluate because often they're generating conversational text for which there's no single "right" answer.
-An imperfect but valuable way to evaluate such applications is to use a second LLM to judge the outputs of the first.
-This can be especially useful if a smaller model is used in the application and a larger, better model is used for evaluation.
+<div id="sdk-content">
+## Pre-built evaluators
 
-## Custom evaluator via SDK
+Pre-built evaluators are a useful starting point setting up evaluations. Refer to [pre-built evaluators](./prebuilt_evaluators) for how to use pre-built evaluators with LangSmith.
 
-For maximal control of evaluator logic, we can write a custom evaluator and run it using the SDK.
+## Create your own LLM-as-a-judge evaluator
+
+For complete control of evaluator logic, create your own LLM-as-a-judge evaluator and run it using the LangSmith SDK ([Python](https://docs.smith.langchain.com/reference/python/reference) / [TypeScript](https://docs.smith.langchain.com/reference/js)).
```

Comment:
not sure how valuable it is to just link to the api ref home page

Reply:
i always need to go searching for the links so my pref is to link to them whenever we mention + link to specific methods when guides are specific enough